### PR TITLE
Doc/store updates

### PR DESCRIPTION
**Pull request by mfisher11**
@alanclayton Last (available) updates to store documentation.

### CODE REVIEW

File: _posts/user/store/2014-12-16-what-is-the-rise-vision-store.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The Rise Vision Store is the area of the platform where you can get and/or purchase the following types of items for your displays:
+
+- **Free widgets**, such as the Text, HTML, Bulletin and Weather widgets, that can be added to your presentations at no cost.
+- **Premium widgets**, such as the Rise Financial Table - Stocks Only widget, that require a monthly subscription fee.
+- **Storage**, if you want to purchase unlimited space for your digital signage content.
+- **Hardware** such as media players to power your digital displays.
+- **Services** such as Content Design and Priority Support.
+- **Training** such as personalized webinars to educate your users about how the platform works.
+
+The Rise Vision Store can be accessed in a number of ways. The quickest way is to click the Store link in the menu at the top of any Rise Vision page. You can also access the Store by adding new content (widgets) to a presentation (See [Add a Widget to your Presentation](http://help.risevision.com/#/user/content/add-a-widget-to-a-presentation)).
```
Comment (alanclayton) [inline _posts/user/store/2014-12-16-what-is-the-rise-vision-store.md]: @mfisher11 I don't think the line about clicking the Store link in the menu at any Rise Vision page is accurate. Let's just update to say the quickest way is to navigate to store.risevision.com

File: _posts/user/store/2015-01-08-how-to-purchase-for-sub-company.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+layout: user-post
+title:  "How to Purchase for a Sub-Company"
+date:   2015-01-08 13:00:00
+category: user/store
+order: 5
+---
+##How to purchase for a sub-company
+In addition to purchasing items for your own company from the Rise Vision Store, you can also purchase items for any one of your sub-companies. To do so, simply switch to the desired sub-company in the Store area of the platform before placing your order.
+
+To switch to a sub-company in the Store area:
```
Comment (alanclayton) [inline _posts/user/store/2015-01-08-how-to-purchase-for-sub-company.md]: This should be formatted as a heading like http://help.risevision.com/#/user/company/add-a-sub-company

File: _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+layout: user-post
+title:  "How to Purchase an Item"
+date:   2015-01-08 11:00:00
+category: user/store
+order: 2
+---
+##How to purchase an item from the Store
+To purchase an item from the Rise Vision Store, navigate to the page of the desired product or service, click its Add to Cart button, and then go to your shopping cart page to complete your purchase.
+
+To navigate to an item:
```
Comment (alanclayton) [inline _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md]: @mfisher11 this should be a heading. See http://help.risevision.com/#/user/company/add-a-sub-company for an example

File: _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Go to the Store by clicking the Store link in the top menu of any Rise Vision page.  (You can also access the Store from the Presentations area of the platform, by adding new content to a placeholder. See [Add a Widget to your Presentation](http://help.risevision.com/#/user/content/add-a-widget-to-a-presentation).)
+2. Find the desired product or service by either browsing the items on the Store's home page, or by using the "Search Products" field at the top of the page. 
+3. Select the item you wish to purchase by clicking on either its image or its price. This takes you to the product page for that item.
+
+
+To add the item to your cart:
```
Comment (alanclayton) [inline _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md]: Should be a heading

File: _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+1. If more than one price option is listed on the product page (for example, payment for a month vs. payment for a year), select the one you want. 
+2. If the item has a Quantity field (shown as "Qty.") and you want to buy more than one, replace the default value (usually "1") with the number of units you want to buy.
+3. When you're ready to purchase the item, click the Add To Cart button. This adds the item to your shopping cart, changes the button into a "Go To Cart" button, and also adds a blue box with a number (indicating the total number of products in your cart) to the shopping cart icon at the top of the page.
+4. If you want to buy other items from the Store before checking out, click the "Continue Shopping" button at the bottom of the product page (or the Store link at the top of the page) to return to the Store's home page.
+
+To go to your shopping cart page:
```
Comment (alanclayton) [inline _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md]: Should be a heading

File: _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+1. When you're ready to check out and ay for your item(s), go to your shopping cart page by clicking on either the "Go To Cart" button at the bottom of an added-to-cart item's product page, or the shopping cart icon at the top of any Store page.  This take you to your shopping cart page.
+2. Review the Bill To and Ship To addresses shown, and click the Edit Address icons (the pencil icons) if you wish to make changes to them.  If your address information is incomplete or incorrect, the problems will be listed in an "Address Validation Error" message in a light red box on this page. Click the Edit Address icon to supply the missing information; doing so will remove the error message and allow you to place the order.
+3. Review the details of your order in the "Review Your Order" area. Here, you can remove an item from your cart by clicking the red circle (with the minus sign icon) to the left of the item's name.  You can also change the quantity of any item that allows multiple units to be purchased.
+
+To place your order:
```
Comment (alanclayton) [inline _posts/user/store/2015-01-08-how-to-purchase-item-from-store.md]: Should be a heading

**Comment by alanclayton** [comment]
@mfisher11 I left a couple comments on the "how to purchase" page. In addition to the changes there, I think we should remove the docs pertaining to sub-companies. Right now, I think it will add too much confusion. 

Also, I think the page, "Store Account Page" should be merged into the "What is Rise Vision Store?" page. What do you think?

**Comment by mfisher11** [comment]
@alanclayton Hopefully this will be the one that is good?

**Comment by alanclayton** [comment]
@mfisher11 looks good.

**Pull request merged**